syntax = "proto3";

message ELG_Parameter {
    int32 ServiceID = 1;
}

message Empty {

}

service Transfer_ELG_Parameter {
    rpc get_ServcieID(Empty) returns(ELG_Parameter);
}
